src/etc/inc/util.inc

```diff
@@ -2474,5 +2488,19 @@ function get_smart_drive_list() {
 	sort($disk_list);
 	return $disk_list;
 }
-
+/* Write the DHCP6 DUID file */
+function write_dhcp6_duid($duidfile) {
+	// Create the hex array from the dhcp6duid config entry and write to file
+				
+	$temp = str_replace(":","",$duidfile);
+	$duid_binstring = pack("H*",$temp);
+	if ($fd = fopen("/var/db/dhcp6c_duid", "wb")){
+		fwrite($fd, $duid_binstring);
+		fclose($fd);
+		return true;
+	}
+	else {
+		return false;
```

Comment:
Maybe it would be nice to check if duid passed on parameter is valid adding a is_duid() call inside this function